### ISSUE TITLE
I got an error trying to grab novel from novelfull.com

### ISSUE THREAD

**Issue by kuakao** [dteviot/WebToEpub#1080]
I got an error trying to grab novel from novelfull.com

I just open this link https://novelfull.com/grandson-of-the-holy-emperor-is-a-necromancer.html in my chrome browser and click WebToEpub extension icon and i got that error.

And when i try to click "Pack EPUB" button there, WebToEpub don't work and no matter how long i wait, it doesn't load any chapters.

Version of the WebToEpub that i used:
WebToEpub 0.0.0.155.Chrome

Thanks

![Clipboard02](https://github.com/dteviot/WebToEpub/assets/88262534/7047cd30-a845-4963-93c8-6a8d26c2d2b1)

![Clipboard01](https://github.com/dteviot/WebToEpub/assets/88262534/80e483d6-356b-4c8c-8105-0b207385ef8d)

**Comment by Naheulf** [comment]
Note 1: Even if it's not the exact same site, I found few domains with _almost_ the same gui:
- [https://novelfull.com/grandson-of-the-holy-emperor-is-a-necromancer.html](https://novelfull.com/grandson-of-the-holy-emperor-is-a-necromancer.html)
- [https://readnovelfull.com/grandson-of-the-holy-emperor-is-a-necromancer.html](https://readnovelfull.com/grandson-of-the-holy-emperor-is-a-necromancer.html)
- [https://readnovelfull.me/grandson-of-the-holy-emperor-is-a-necromancer/](https://readnovelfull.me/grandson-of-the-holy-emperor-is-a-necromancer/)
- [https://novelfullbook.com/novel-full-book/grandson-of-the-holy-emperor-is-a-necromancer](https://novelfullbook.com/novel-full-book/grandson-of-the-holy-emperor-is-a-necromancer)
- [https://novelnext.com/novelnext/grandson-of-the-holy-emperor-is-a-necromancer](https://novelnext.com/novelnext/grandson-of-the-holy-emperor-is-a-necromancer)
- [https://pandasnovel.net/pandasnovel/grandson-of-the-holy-emperor-is-a-necromancer](https://pandasnovel.net/pandasnovel/grandson-of-the-holy-emperor-is-a-necromancer)

Note 2: They are not all up to date with the latest chapter.
Note 3: As all those sites are pretty similar, does it worth it to create a common parser like the WordPress one?

**Comment by dteviot** [comment]
@Naheulf

Ahem.
https://github.com/dteviot/WebToEpub/blob/e964cffe608e287afc22d6ded30a570ff24400dc/plugin/js/parsers/NovelfullParser.js#L3-L16

Feel free to add the missing ones.

**Comment by dteviot** [comment]
@kuakao 

Content node id changed from "chr-content" to "chapter-content".  Easy fix. 

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://novelfull.com/grandson-of-the-holy-emperor-is-a-necromancer.html, chapters 1 &amp; 2

For my notes: 22 minutes work (mostly because I got side tracked seeing what happened to the other sites.)
